src/DependencyInjection/Configuration.php

```diff
@@ -142,6 +142,10 @@ function (string $value): string {
                             ->prototype('scalar')->end()
                             ->defaultValue(['jpg', 'jpeg', 'gif', 'png', 'tif', 'tiff', 'bmp', 'svg', 'svgz'])
                         ->end()
+                        ->scalarNode('imagine_service')
+                            ->info('By default Contao tries to take the best Imagine service out of Gmagick, Imagick and Gd (in this order). If you would like to use a different service, configure the service id here.')
```

Comment:
When will this be displayed?

Reply:
`config:dump-reference`  :)

Reply:
Nice. However we don't have this for any other parameter, therefore I would prefer to remove this from this PR and to create a separate PR which adds descriptions for all parameters. WDYT?

Reply:
Nah, it belongs to this PR. Just add the info in another commit :)